### ISSUE TITLE
[Keyboard Navigation - Make Code - Header] Focus indicator is not clearly visible on Header elements.

### ISSUE THREAD

**Issue by Deepika-0890** [microsoft/pxt-microbit#2366]
**User Experience:**
User will find difficulty to see any control if keyboard focus is not visible on any interactive control as they won't be able to take an action on such control and also they will surely skip the functionality and will proceed further

**Test Environment:**
Version 1903 (OS Build 18867.1000)
Platform: Edge, Chrome

**Repro Steps:**
1. Launch application URL : https://makecode.microbit.org/beta
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Navigate to Header elements.
5. Verify, if focus indicator is clearly visible or not under Header controls.

**Actual Result:**
Focus indicator is not clearly visible on 'Share', 'Help' and 'More' header elements, due to this visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Focus indicator should clearly visible via dotted line or highlighted control on 'Share', 'Help' and 'More' header elements, due to this visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Note:**
1. This issue also exists with Chrome.
2. Same issue exists under 'Share Project' for 'Publish Project' and 'Share Project' pop up windows.

**MAS Reference:**
MAS 2.4.7 - Focus Visible

**Accessibility Insights!** - Identify accessibility bugs before check-in and make bug fixing faster and easier.
[MAS2.4.7_Focus visibility is very low.zip](https://github.com/microsoft/pxt-microbit/files/3476678/MAS2.4.7_Focus.visibility.is.very.low.zip)

[MAS2.4.7_Focus visibility is low.zip](https://github.com/microsoft/pxt-microbit/files/3476700/MAS2.4.7_Focus.visibility.is.low.zip)

**Comment by jwunderl** [comment]
Issue is that it just darkens the buttons instead of giving them a separate outline; @Deepika-0890 besides the highlight color issue (https://github.com/microsoft/pxt-microbit/issues/2375), does high contrast mode successfully indicate these buttons when focused like this:

![Screen Shot 2019-08-08 at 11 32 45 AM](https://user-images.githubusercontent.com/5615930/62728309-4f858f80-b9d0-11e9-8284-a3d1f21da0f0.png)

High contrast mode is the mode intended to be accessible for visually impaired users.
